layouts/compliance/cis.html

```diff
@@ -0,0 +1,224 @@
+{{ define "main" }}
+
+{{ partial "hero.html" (dict "title" .Title) }}
+<div class="container mx-auto">
+    <section class="px-4 md:px-0">
+        <div class="container mx-auto max-w-4xl text-center my-16 px-6">
+            <h2>{{ .Title }}</h2>
+            <p>
+                {{ .Params.description }}
+            </p>
+            <p>
+                Pulumi can assist with making your {{ .Params.cloud }} cloud infrastructure meet CIS benchmarks.
+            </p>
+            <div class="mt-10">
+                <a href="#sa-form" rel="noopener noreferrer" class="btn-primary">Speak with a Solutions Architect!</a>
```

Comment:
I think we want to remove an exclamation mark here

Reply:
I think we should have two CTAs, the left one should say "Speak with an Architect" and the right one that is the secondary CTA should say "Get Started with Pulumi"